examples/parallel_streaming_decryptor.rs

```diff
@@ -0,0 +1,41 @@
+use bytes::Bytes;
+use rayon::prelude::*;
+use self_encryption::{deserialize, streaming_decrypt_from_storage, DataMap, Error, Result};
+use std::{fs::File, io::Read, path::Path};
+use xor_name::XorName;
+
+fn main() -> Result<()> {
+    // Load the data map from file or another source
+    let data_map = load_data_map("path/to/data_map")?;
```

Comment:
Does this example run?

Reply:
Thanks @grumpy I have added clap and made this use cmd line args now